package.json

```diff
@@ -2,15 +2,30 @@
   "name": "vote.gov",
   "version": "1.0.0",
   "main": "gulpfile.js",
+  "config": {
+    "hugo": {
+      "en": "config.toml",
+      "es": "config_es.toml"
+    },
+    "urls": {
+      "en": {
+        "staging": "https://vote-gov-staging.apps.cloud.gov/",
+        "production": "https://vote-gov.apps.cloud.gov/"
+      },
+      "es": {
+        "staging": "https://vote-gov-staging.apps.cloud.gov/es/",
+        "production": "https://vote-gov.apps.cloud.gov/es/"
+      }
+    }
+  },
```

Comment:
It makes sense to have the `hugo` config in here, but I think it'd be better to move the `urls` config elsewhere, because:

1. It's related to deployment rather than building, so shouldn't be tied so directly to the package data.
2. This use of `npm config` for _custom_ package variables (rather than the standard `npm` ones) seems to be undocumented, even though it works. (Though maybe it's a common thing? I don't touch `npm` very often, so feel free to correct me!) And that's probably OK, but having a config key as generic as `urls` may be asking for future collision.

Possible solutions:

1. Move all the custom config under a single key that's less likely to collide (e.g. `votegov`)
2. Move deployment config out to `circle.yml`. (Trouble is, this makes building on a dev machine a little more complicated.)

Just thinking aloud here - what do you think?